frontend/src/features/settings/components/creditAccount/AccountDetails.tsx

```diff
@@ -98,6 +98,8 @@ export const AccountDetails = () => {
         setShowCloseCreditAccountModal(false);
         handleMenuClose();
         refetchCreditAccount();
+      } else {
+        console.error(status);
```

Comment:
There's no point in printing the status with console.error. Try printing a relevant error message like "Updating credit account status failed or something". This is mostly for dev purposes, showing what action has failed.